token/program-2022/src/extension/confidential_transfer/processor.rs

```diff
@@ -454,55 +457,61 @@ fn process_transfer(
         // The zero-knowledge proof certifies that:
         //   1. the transfer amount is encrypted in the correct form
         //   2. the source account has enough balance to send the transfer amount
-        let proof_context = verify_transfer_proof(
+        let maybe_proof_context = verify_transfer_proof(
             account_info_iter,
             proof_instruction_offset,
             split_proof_context_state_accounts,
+            no_op_on_uninitialized_split_context_state,
+            close_split_context_state_on_execution,
             source_decrypt_handles,
         )?;
+        // If `maybe_proof_context` is `None`, then this means that
+        // `close_split_context_state_on_execution` is true and a required context state account is
```

Comment:
This should be `no_op_on_uninitialized_split_context_state`, right?

Reply:
Thanks for the catch! Fixed.